x-pack/plugins/observability_solution/apm/server/routes/errors/get_error_groups/get_error_group_sample_ids.ts

```diff
@@ -66,16 +67,18 @@ export async function getErrorGroupSampleIds({
           should: [{ term: { [TRANSACTION_SAMPLED]: true } }], // prefer error samples with related transactions
         },
       },
-      _source: [ERROR_ID, 'transaction'],
+      fields: [ERROR_ID],
```

Comment:
If I'm not mistaken, the 'transaction' was added for the scoring, so I think the field `TRANSACTION_SAMPLED`, should we also added not sure why just 'transaction' was in this case
@dgieselaar can you confirm?